packages/usa-in-page-navigation/src/usa-in-page-navigation.stories.js

```diff
@@ -18,4 +20,26 @@ export const TestCustomContentSelector = TestCustomContentTemplate.bind();
 TestCustomContentSelector.args = {
   customContentSelector: true,
 };
+
+export const TestCustomHeaderSelector = TestCustomHeaderTemplate.bind();
+TestCustomHeaderSelector.argTypes = {
+  includeHeaders: {
+    defaultValue: "All",
+    name: "Include these headers in link list",
+    options: [
+      "Default value",
+      "All",
+      "h1",
+      "h2",
+      "h3",
+      "h4",
+      "h5",
+      "h6",
+      "Error in value",
+      "Empty value",
```

Comment:
Removed this option in [8eabccc](https://github.com/uswds/uswds/pull/5444/commits/8eabccc76cfcec8591c25e40aaf737324aee2d29) since it is covered by "default".